test/exercises/c-ray/old/c-ray.badscenes.chpl

```diff
@@ -1 +0,0 @@
-require "c-ray.chpl";
```

Comment:
`require` alone does not cause the modules brought in to not be dead (per the discussion in #25112). So, the old version would be equivalent to an empty source file (and do nothing useful).

Since `c-ray.chpl` creates the implicit module `module c-ray` which has a name that is not a valid identifier, I can't `use c-ray;`. Changing it to a symlink was the smallest change I could think of to get it working.

I don't think `require` is particularly relevant for what this test is doing. I think it was just a way to not `use` c-ray.chpl since it has an unfriendly name.